src/sentry/static/sentry/app/components/stream/groupChart.jsx

```diff
@@ -25,8 +26,12 @@ class GroupChart extends React.Component {
 
   render() {
     const stats = this.props.statsPeriod
-      ? this.props.data.stats[this.props.statsPeriod]
+      ? this.props.hasDynamicIssueCounts && this.props.data.filtered
+        ? this.props.data.filtered.stats[this.props.statsPeriod]
+        : this.props.data.stats[this.props.statsPeriod]
```

Comment:
Can we destructure from `this.props`, it'll make it a bit easier to read.

Reply:
Will do